install.sh

```diff
@@ -55,68 +36,30 @@ function install_seed_voicecard_driver()
     cd seeed-voicecard
     sudo ./install.sh
     cd ..
-    mv seeed-voicecard ~/seeed-voicecard
-}
-
-function install_flite_from_source()
-{
-    wget http://www.festvox.org/flite/packed/flite-2.0/flite-2.0.0-release.tar.bz2
-    tar xf flite-2.0.0-release.tar.bz2
-    cd flite-2.0.0-release
-    ./configure
-    make
-    sudo make install
-    cd ..
-    rm -rf flite-2.0.0-release*
+    tar czf ~/seeed-voicecard.tar.gz seeed-voicecard
+    rm -rf seeed-voicecard
 }
 
 function install_dependencies()
 {
     install_seed_voicecard_driver
-    if /usr/bin/dpkg --search /usr/bin/dpkg
-    then
-        sudo -E apt install -y libatlas-base-dev
-    else
-        return 1;
-    fi
 }
 
-function install_snowboy()
-{
-    if install_dependencies
-    then
-        root_dir=$(pwd)
-        sudo pip3 install git+https://github.com/Kitt-AI/snowboy.git
-        if [ $? -ne 0 ]; then
-            echo "FAILED: Unable to make Snowboy Detect file. Please follow manual instructions at https://github.com/kitt-AI/snowboy"
-            echo "You may also use PocketSphinx Detector if you are unable to install snowboy on your machine"
-        else
-            echo "Snowboy Detect successfully installed"
-        fi
-        cd "$root_dir"
-        rm -rf snowboy
-    else
-        echo "FAILED: Installation of Snowboy on your system is not supported presently. Please follow manual instructions at https://github.com/kitt-AI/snowboy"
-    fi
-}
-
-
-
 function susi_server(){
     if  [ ! -d "susi_server" ]
     then
         mkdir $DIR_PATH/susi_server
         cd $DIR_PATH/susi_server
-        git clone https://github.com/fossasia/susi_server.git
+        git clone --recurse-submodules https://github.com/fossasia/susi_server.git
         git clone https://github.com/fossasia/susi_skill_data.git
+        # The .git folder is big. Delete it (we don't do susi_server deveplopment here, so no need to keep it)
```

Comment:
@stealthanthrax We will get update by different method. I expect *susi_server* is released by version, and we will download it by version (not continuous update).

I also expect `susi_server` to be built and released in form of Debian package, so that we don't have to install `gradle` tool, other Java libraries just to build it on RPi.

For now, when *susi_server* is not provided in the form I want, we will include a fixed version of *susi_server* in each version of SD card image. In case the developer wants to update it without reflashing RPi with new SD image, he can just download tarball of *susi_server*.

Reply:
Okay @hongquan , makes sense 👍